<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditCoffeeForm</class>
 <widget class="QWidget" name="AddEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактировать кофе</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="placeholderText">
      <string>Название сорта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="roastLineEdit">
     <property name="placeholderText">
      <string>Степень обжарки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="grindLineEdit">
     <property name="placeholderText">
      <string>Молотый/в зернах</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="descriptionLineEdit">
     <property name="placeholderText">
      <string>Описание вкуса</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="priceLineEdit">
     <property name="placeholderText">
      <string>Цена</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="volumeLineEdit">
     <property name="placeholderText">
      <string>Объем упаковки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
